### PR TITLE
MAINT: runtests help text cleanup

### PR DESCRIPTION
**Pull request by chtrac**
Solves #18804.

I tried to make the help text for the verbosity flag to runtests.py better explain what it's actually doing. Please come with feedback if you think the new text could be made better.

I also noticed that the casing of the help texts for the flags in runtests.py were inconsistent, so I made them all sentence cased (which is what most already were).

### CODE REVIEW

File: runtests.py [hunk 1]
```diff
@@ -74,40 +74,44 @@
 def main(argv):
     parser = ArgumentParser(usage=__doc__.lstrip())
     parser.add_argument("--verbose", "-v", action="count", default=1,
-                        help="more verbosity")
+                        help="Add one verbosity level to pytest. Default is 0")
     parser.add_argument("--debug-info", action="store_true",
-                        help=("add --verbose-cfg to build_src to show compiler "
-                              "configuration output while creating "
+                        help=("Add --verbose-cfg to build_src to show compiler"
+                              " configuration output while creating "
```
Comment (eric-wieser) [inline runtests.py:80]: By convention we put the space on the end of the previous line, as things were before you changed this line

Comment (cdahlin) [reply runtests.py:80]: Does this convenience override E501 (line length), or should it fulfill both (so I should move compiler to the next line)?

Comment (charris) [reply runtests.py:80]: Just break the line into shorter segments.

**Comment by charris** [comment]
Thanks @cdahlin .

**Pull request merged**